test/e2e/all_in_one_test.go

```diff
@@ -73,7 +80,44 @@ func TestDeployAllInOneDBLESS(t *testing.T) {
 	defer cancel()
 
 	t.Log("building test cluster and environment")
-	builder := environments.NewBuilder().WithAddons(metallb.New())
+	addons := []clusters.Addon{}
+	addons = append(addons, metallb.New())
+	if b, err := loadimage.NewBuilder().WithImage(os.Getenv(imageOverrideEnvVar)); err == nil {
```

Comment:
might be more streamlined to read the variable once, globally:

instead of 

```go
var imageOverrideEnvVar = "TEST_KONG_CONTROLLER_IMAGE_OVERRIDE"
```

try

```go
var imageOverride = os.Getenv("TEST_KONG_CONTROLLER_IMAGE_OVERRIDE")
```

Reply:
8a95d726